src/graphql_schema.ts

```diff
@@ -9,38 +11,35 @@ import {
 
 import InputUnionType from './input_union_type';
 
-const MAX_INT = 2147483647;
-const MIN_INT = -2147483648;
-const coerceIntString = (value: number | string) => {
-    if (Array.isArray(value)) {
-        throw new TypeError(`IntString cannot represent an array value: [${String(value)}]`);
-    }
-    if (Number.isInteger(value)) {
-        if (value < MIN_INT || value > MAX_INT) {
-            throw new TypeError(
-                `Value is integer but outside of valid range for 32-bit signed integer: ${String(
-                    value
-                )}`
-            );
-        }
-        return value;
-    }
-    return String(value);
-};
-
 /** @see https://stackoverflow.com/a/49911974 */
-const intString = new GraphQLScalarType({
-    name: 'IntString',
-    serialize: coerceIntString,
-    parseValue: coerceIntString,
+// tslint:disable-next-line: variable-name
+const FilterValue = new GraphQLScalarType({
```

Comment:
Perform no coercion/casting in the graphql layer. Stricten the available input types for filter value.